generator/routing_helpers.cpp

```diff
@@ -39,7 +39,10 @@ bool ParseOsmIdToFeatureIdMapping(string const & osmIdsToFeatureIdPath,
   }
 
   osmIdsToFeatureIds.ForEach([&](gen::OsmID2FeatureID::ValueT const & p) {
-    AddFeatureId(osmIdToFeatureId, p.second /* feature id */, p.first /* osm id */);
+    if (p.first.IsWay())
+    {
```

Comment:
I added these to make the block identifiable wrt the function end. Despite redundancy, they increase readability, in my opinion.